phpunit.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/7.5/phpunit.xsd"
+         bootstrap="vendor/autoload.php"
+         forceCoversAnnotation="true"
+         beStrictAboutCoversAnnotation="true"
```

Comment:
non-blocking question: the `forceCoversAnnotation` and `beStrictAboutCoversAnnotation` might not be necessary. what do you think? they can improve the quality of coverage reporting (eg. incidentally executed code that a parent method calls won't count toward coverage reporting totals) but may be a bit of a hindrance getting things rolling to start with, explaining that context about what the `@covers` annotations do for the first tests in the tutorial would be a heavy burden for a first-time tester